content/ja/docs/setup/production-environment/container-runtimes.md

```diff
@@ -235,7 +235,6 @@ CRI-Oはデフォルトでsystemd cgroupドライバーを使用し、おそら
 [crio.runtime]
 conmon_cgroup = "pod"
 cgroup_manager = "cgroupfs"
-```
```

Comment:
I don't think this can be removed?

Reply:
Thank you for your comments, I've fixed it.